Source/Samples/Sections/Widgets/EntrySection.cs

```diff
@@ -66,5 +69,100 @@ public EntrySection()
             return ("Invisible text entry:", entry);
         }
 
+        public (string, Widget) CreateSearchEntry()
+        {
+            var entry = new Entry();
```

Comment:
Gtk actually has a Gtk.SearchEntry, rename this to something like custom entry actions section.